examples/other_languages.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import os
+from subprocess import PIPE, Popen
+
+# insert your client_token into sigopt_creds.py
+# otherwise you'll see "This endpoint requires an authenticated user" errors
+from sigopt_creds import client_token
+
+from sigopt.interface import Connection
+
+class SubProcessEvaluator(object):
+  def __init__(self, filename):
+    assert os.path.isfile(filename)
+    self.filename = filename
+
+  # Take a suggestion from sigopt and evaluate your function
+  # Sends command line arguments to your executable file with the same names as the
+  # parameters of your experiment. Expected output is one line containing a float that
+  # is your function evaluated at the suggested assignments.
+  # For example, if your filename is test and you have one double parameter with suggested
+  # value 11.05, this script will run
+  #         ./test --x=11.05
+  def evaluate_metric(self, assignments):
+    arguments = [
+      '--{}={}'.format(param_name, assignment)
+      for param_name, assignment
+      in assignments.to_json().iteritems()
+    ]
+    process = Popen(['./{}'.format(self.filename)] + arguments, stdout=PIPE)
```

Comment:
can `filename` just be the full path? that way it can also be something on the path

and there might be additional arguments too

so can the user say

    ./run_sigopt --command "run_command --very-slowly" --experiment_id 1 --client_token X